app/helpers/model_helpers.rb

```diff
@@ -0,0 +1,17 @@
+module ModelHelpers
```

Comment:
@Laszlo-JFLMTCO @akintner i think you could create a module or a PORO in your models directory. agreed - it's not super clear that they are intended for your views. it's pretty subtle - but they are automatically available in every view template you write, but you have to manually include them everywhere else you want to use them